test/fixtures/full_config.yml

```diff
@@ -138,15 +138,15 @@ Bundler/InsecureProtocolSource:
   Description: The source `:gemcutter`, `:rubygems` and `:rubyforge` are deprecated
     because HTTP requests are insecure. Please change your source to 'https://rubygems.org'
     if possible, or 'http://rubygems.org' if not.
-  Enabled: false
+  Enabled: true
   VersionAdded: '0.50'
   Include:
   - "**/*.gemfile"
   - "**/Gemfile"
   - "**/gems.rb"
 Bundler/OrderedGems:
   Description: Gems within groups in the Gemfile should be alphabetically sorted.
-  Enabled: false
```

Comment:
While many projects have somewhat disorganized Gemfiles, many projects have tried to organize their Gemfiles into sections by what type of Gem it is (e.g. grouping gems that pertain to datastores).

While this rule could be disabled on a per-repo basis, is our "new" stance that we prefer organizing alphabetically over by category?

Reply:
My experience with this cop is that it will only require gems to be sorted within "sections", so if those sections are separated by a blank line (or even a comment based on the default configuration), it will only require ordering within those sections. "group" in the description of the cop is generic and does not refer to the Gemfile `group` method.

We may want to consider changing the configuration for `TreatCommentsAsGroupSeparators` to false.

With the default true setting we can get a strange ordering:
```ruby
gem "a"
gem "s"
gem "x"
# comment on "b"
gem "b"
gem "t"
gem "y"
```

If it is set to false, then the auto-correct is smart enough to move comments with the gem that they precede.